mobile-app/app/components/OceanInterface/TransactionDetail.tsx

```diff
@@ -2,48 +2,65 @@ import { View } from 'react-native'
 import { tailwind } from '@tailwind'
 import { translate } from '@translations'
 import { openURL } from '@api/linking'
-import { ThemedActivityIndicator, ThemedIcon, ThemedText } from '@components/themed'
+import { ThemedActivityIndicator, ThemedIcon, ThemedTextV2, ThemedViewV2 } from '@components/themed'
 import { TransactionCloseButton } from './TransactionCloseButton'
 import { TransactionIDButton } from './TransactionIDButton'
+import { useThemeContext } from '@shared-contexts/ThemeProvider'
 
 interface TransactionDetailProps {
   broadcasted: boolean
   txid?: string
   txUrl?: string
   onClose: () => void
   title?: string
+  transactionStatusCode?: number
 }
 
 export function TransactionDetail ({
   broadcasted,
   txid,
   txUrl,
   onClose,
-  title
+  title,
+  transactionStatusCode
 }: TransactionDetailProps): JSX.Element {
   title = title ?? translate('screens/OceanInterface', 'Broadcasting...')
+
+  const { isLight } = useThemeContext()
+
+  const themedOrange = isLight ? 'border-warning-500' : 'border-darkwarning-500'
```

Comment:
can make use of tailwind directly + themed component to lessen the ternary operator logic
``` light={tailwind('bg-mono-light-v2-00', {
        'border-warning-500': <condition_here>
        'border-success-500': <condition_here>
      })}

Reply:
@pierregee done try check hahaa